### ISSUE TITLE
Planetscale driver: unsupported column type NULL

### ISSUE THREAD

**Issue by JClackett** [prisma/prisma#21369]
Getting this error when running a raw query

```
Error: Unsupported column type: NULL
    at fieldToColumnType (/var/task/node_modules/@prisma/adapter-planetscale/dist/index.js:79:13)
    at /var/task/node_modules/@prisma/adapter-planetscale/dist/index.js:118:42
    at Array.map (<anonymous>)
    at PrismaPlanetScale.queryRaw (/var/task/node_modules/@prisma/adapter-planetscale/dist/index.js:118:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /var/task/node_modules/@prisma/client/runtime/library.js:26:4983 {
  clientVersion: '5.4.1'
```

query is:

```mysql
      SELECT
        Spot.id, Spot.name, Spot.type, Spot.address, null as image, null as blurHash,
        (SELECT AVG(rating) FROM Review WHERE Review.spotId = Spot.id) AS rating,
        (CAST(COUNT(ListSpot.spotId) as CHAR(32))) AS savedCount
      FROM
        Spot
      LEFT JOIN
        ListSpot ON Spot.id = ListSpot.spotId
      WHERE
        Spot.creatorId = ? AND Spot.deletedAt IS NULL AND (Spot.creatorId = ? OR (Spot.publishedAt IS NULL OR Spot.publishedAt < NOW()))
      GROUP BY
        Spot.id
      ORDER BY
        Spot.createdAt DESC, Spot.id
      LIMIT ?
      OFFSET ?;
```

**Comment by aqrln** [comment]
Thanks for the report @JClackett! We need the schema to be able to reproduce this, could you please post it here as well?

**Comment by aqrln** [comment]
@JClackett btw, are you using a `$queryRaw`, or did you post an SQL statement generated by some Prisma query? I don't think it should matter either way for the reproduction since the error happens when decoding the response in the driver adapter, but asking just in case.

**Comment by JClackett** [comment]
Yeah using queryRaw!

**Comment by JClackett** [comment]
just thinking about it, could it because im creating "null" columns manually in the query?

```sql
Spot.id, Spot.name, Spot.type, Spot.address, null as image, null as blurHash,
```

Im doing this because I then populate the result with each spots latest image, and wanted to make sure that the resulting query had the keys available (they are just unknown at that point)

**Comment by aqrln** [comment]
I reproduced the issue, should be an easy fix.

**Comment by aqrln** [comment]
@JClackett can you check if `@prisma/adapter-planetscale@5.4.1-pre` fixes the issue for you?

**Comment by JClackett** [comment]
appears to be working!

**Comment by JClackett** [comment]
Thanks for the unbelievably fast response, just deployed it live and she's purring 🚀

**Comment by aqrln** [comment]
Due to driver adapters releases not having been properly automated before 5.4.2 and a mishap with a backport PR not having been merged to the release branch before the patch release, this issue is fixed in 5.4.1 but not in 5.4.2. The fix will be included again in 5.5.0.

Sorry about this, it will not happen again since now the driver adapter packages now use the same publishing mechanism as the rest of Prisma packages.

Although generally the versions of all Prisma packages you use should match, in this specific instance it's totally fine to use `@prisma/adapter-planetscale@5.4.1` together with `prisma@5.4.2` and `@prisma/client@5.4.2` to avoid this issue.
